### ISSUE TITLE
"name" option not working as expected

### ISSUE THREAD

**Issue by Amaimersion** [josdejong/jsoneditor#769]
```javascript
const options = {
    mode: "view",
    name: "Device Information"
};
const editor = new JSONEditor(
     <element>,
     options,
     null
);
```
In the earlier version it has been working perfectly: name of the root was "Device Information". But after 6.1.0 name of the root is always `object` instead of "Device Information". 

Calling of `editor.setName("Device Information")` before `editor.set(data);` don't make anything, but calling of `editor.setName("Device Information")` after `editor.set(data);` sets root name to "Device Information".

**Comment by josdejong** [comment]
Thanks for reporting Sergey, there was a regression introduced in `v6.1.0` which broke this feature. Fixed via 174918f.

**Comment by josdejong** [comment]
Fixed now in `v6.3.0`
